modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -35,10 +35,35 @@
 #include <openspace/scripting/scriptengine.h>
 #include <ghoul/filesystem/filesystem.h>
 #include <ghoul/logging/logmanager.h>
+#include <scn/scn.h>
 
 namespace {
-    constexpr std::string_view _loggerCat = "SkyBrowserModule";
+constexpr std::string_view _loggerCat = "SkyBrowserModule";
+
+bool browserBelongsToCurrentNode(std::string identifier) {
+    std::string delimiter = "_";
+    size_t found = identifier.find(delimiter);
```

Comment:
Just a tiny bit faster if `size_t found = identifier.find('-');` rather than "-" since '-' is guaranteed to be just a single character